debug.log

```diff
@@ -0,0 +1,6 @@
+[0103/171014.750:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)
```

Comment:
I don't think you wanted that file to be committed :D

Reply:
yes indeed

Reply:
Maybe add .log to the local .gitignore, or any that could be affected. Makes sense anyway.